Solution to 0x10-python-network_0 task
